### PR TITLE
Issue #1553 feat : Remove and Edit topic and subtopic of a session - …

### PR DESCRIPTION
**Pull request by suvarnakale**
…done

### CODE REVIEW

File: src/components/SelectTopic.tsx [hunk 1]
```diff
@@ -58,7 +68,7 @@ const SelectTopic: React.FC<TopicSubtopicProps> = ({
           <Select
             labelId="topic-select-label"
             id="topic-select"
-            value={selectedTopic}
+            value={selectedTopics || selectedTopic}
```
Comment (itsvick) [inline src/components/SelectTopic.tsx:71]: @suvarnakale why to variables for selectedTopic

File: src/components/SessionCardFooter.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ const SessionCardFooter: React.FC<SessionCardFooterProps> = ({
   const [eventStatus, setEventStatus] = React.useState('');
 
   const EventDate = getDayMonthYearFormat(item?.startDateTime);
-
+  var removeTopic = false;
```
Comment (itsvick) [inline src/components/SessionCardFooter.tsx:50]: @suvarnakale avoid using var

File: src/components/SessionCardFooter.tsx [hunk 3]
```diff
@@ -61,7 +61,7 @@ const SessionCardFooter: React.FC<SessionCardFooterProps> = ({
         });
 
         const courseData = response?.result?.data[0];
-        let courseId = courseData._id;
+        let courseId = courseData._id || '66daea0246cec8001342a6ce';
```
Comment (itsvick) [inline src/components/SessionCardFooter.tsx:64]: @suvarnakale remove this hard coded id

File: src/components/SelectTopic.tsx (hunk 1 above)
Comment (suvarnakale) [reply src/components/SelectTopic.tsx:71]: okay.. I have added this in setter function.. will remove from here.

File: src/components/SessionCardFooter.tsx (hunk 2 above)
Comment (suvarnakale) [reply src/components/SessionCardFooter.tsx:50]: ok.. will use let then

File: src/components/SessionCardFooter.tsx (hunk 3 above)
Comment (suvarnakale) [reply src/components/SessionCardFooter.tsx:64]: ok

**Pull request merged**